multus-cni.yaml

```diff
@@ -61,8 +61,10 @@ subpackages:
     pipeline:
       - runs: |
           mkdir -p ${{targets.contextdir}}/usr/src/multus-cni/bin
+          ln -sf /usr/bin/multus ${{targets.contextdir}}/usr/src/multus-cni/bin/multus
```

Comment:
I think we need an install_multus binary too for the whereabouts chart. https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/deployments/multus-daemonset.yml

Reply:
done.